docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -81,6 +81,16 @@ spec:
       admin:
         mit:
           kadminServer: krb5-kdc
+        # or...
+        activeDirectory:
+          # ldapServer must match the AD Domain Controller's FQDN or GSSAPI authn will fail
+          # You may need to set AD as your fallback DNS resolver in your Kube DNS Corefile
+          ldapServer: addc.example.com
+          passwordCacheSecret:
```

Comment:
Can we please also document
```
          ldapTlsCaSecret:
            # namespace: default
            name: secret-operator-ad-ca
```
Also that we will always use ldaps and the Secret needs the key `ca.crt`

Reply:
77c0c05bffc39051d0dedcb5af108fa9243b2c7d